src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/SAPDownloadedPackageTests.swift

```diff
@@ -28,7 +28,7 @@ final class SAPDownloadedPackageTests: XCTestCase {
 		let bytes = [0xA, 0xB, 0xC, 0xD]
 		// The bin and signature were  made for different data sets
 		let package = try SAPDownloadedPackage.makePackage(bin: Data(bytes: bytes, count: 4),
-														   signature: try SAPDownloadedPackage.makeSignature(data: Data(bytes: bytes, count: 3), key: signingKey).asList() //swiftlint:disable:this vertical_parameter_alignment_on_call
+														   signature: try SAPDownloadedPackage.makeSignature(data: Data(bytes: bytes, count: 3), key: signingKey).asList() // swiftlint:disable:this vertical_parameter_alignment_on_call
```

Comment:
The linter warnings are already fixed on the release branch, it's probably the easiest to just revert that commit and update your branch from the release branch.

Reply:
There was only this change marked as conflict ;-) so easy merging ;-)